.github/workflows/unit-testing-and-coverage.yml

```diff
@@ -39,14 +39,14 @@ jobs:
       - name: "flake8 Style Checking"
         # language=bash
         run: |
-          pip install colorama flake8
+          pip install colorama flake8 flake8-bugbear
           python -m flake8 aiosmtpd setup.py housekeep.py release.py
       - name: "Docs Checking"
         # language=bash
         run: |
           pip install colorama pytest pytest-mock sphinx sphinx-autofixture sphinx_rtd_theme
-          pytest -v aiosmtpd/docs
-          python setup.py build_sphinx
+          sphinx-build -b doctest -d build/.doctree aiosmtpd/docs build/doctest
+          sphinx-build -b html    -d build/.doctree aiosmtpd/docs build/html
```

Comment:
`pytest` doctest can't conditionally skip blocks.

So we now use Sphinx' own `sphinx.ext.doctest`.